shared/data_tools_core_minio.py

```diff
@@ -31,8 +31,14 @@ def build_secure_url(self, blob_name: str, expiration_offset: int = None, bucket
         signed_url = super().build_secure_url(blob_name = blob_name,
                                               expiration_offset = expiration_offset,
                                               bucket = bucket)
-        return signed_url
+        if settings.DOCKER_COMPOSE_CONTEXT == True:
+            # note need to proxy request through local dispatcher so users can access from the browser
```

Comment:
replace minio url with a url path to a proxy if `DOCKER_COMPOSE_CONTEXT` is true